### ISSUE TITLE
Old machines

### ISSUE THREAD

**Issue by Tarkov** [zmaster587/AdvancedRocketry#283]
At some point you changed some recipes. Since then I have not been able to use any machine in my oldest game. I tried to build them from zero, making new plugs and inputs. There is always a red cross. Is there any way I can do it?

**Comment by zmaster587** [comment]
Which machine is it or is it all of them?

**Comment by Tarkov** [comment]
All of them stopped working. They were working for a while but when I touched them stopped. I tried rebuilding the Lathe with all new components and nothing. I think the electrolyzer and the chemical reactor are still working because I did not touch them, but not sure because I do not want to manipulate them too much.

**Comment by zmaster587** [comment]
That's odd, the lathe has never had it's structure changed.  Nor do any of the blocks store data once broken.  Would it be possible to take a few screenshots of the machine, preferably one containing the red cross?

**Comment by Tarkov** [comment]
![2016-11-12_11 19 30](https://cloud.githubusercontent.com/assets/19404111/20237168/4c46d3ba-a8cb-11e6-9762-7ff3bac8de22.png)

![2016-11-12_11 19 59](https://cloud.githubusercontent.com/assets/19404111/20237174/67326bc6-a8cb-11e6-92f6-84ae11b19475.png)

![2016-11-12_11 25 33](https://cloud.githubusercontent.com/assets/19404111/20237177/7e2891d4-a8cb-11e6-86d6-59dc9afeb4bd.png)

I tried all the combinations with the orientation of the motor and hatches. Also the output hatch has the image of the input hatch in the holo projection.

The same happens with the rolling machine.

![2016-11-12_11 24 21](https://cloud.githubusercontent.com/assets/19404111/20237185/c4bdeb6c-a8cb-11e6-8dd1-6ec340b8bc54.png)

![2016-11-12_11 24 41](https://cloud.githubusercontent.com/assets/19404111/20237190/d2cdc894-a8cb-11e6-8e33-f7ba3da80bb4.png)

![2016-11-12_11 25 10](https://cloud.githubusercontent.com/assets/19404111/20237204/04ac99bc-a8cc-11e6-8bf9-994a05023a82.png)

**Comment by zmaster587** [comment]
Very odd indeed, the locations indicated should actually be air blocks.  Would it be at all possible to make a backup and try with only libvulpes and AR installed.  If it's still broken with that would it be possible to upload a copy of the level.dat, advRocketry/temp.dat and the region file with the machines?

**Comment by Tarkov** [comment]
I found out a couple of things. I have two different id numbers for the fluid output hatches. The old ones and the new ones. Obviously the old ones dont work, so I replaced them. Also the main problem comes from "Flood Lights mod" I have a room with a lot of these lights, when they are switched off everything works right, but when the lights are on the machines dont work even if I rebuild them and the red crosses appear. I think is related with the render and the type of the lights. 

It seems like it is not a problem with Advanced Rocketry but with this "lights mod". So I am changing the lights.

Thank you very much for helping me with this weird issue.
